firmware/gerrit_watcher/gerrit_watcher.ino

```diff
@@ -13,10 +15,13 @@ const auto PASSWORD = "your_password";
 const String GERRIT_URL = "http://your_gerrit_url:8080";
 const auto GERRIT_USERNAME = "your_gerrit_username";
 const auto GERRIT_HTTP_PASSWORD = "your_gerrit_http_password";
+const auto MY_EFFECT = Effect::PULSE;
 #else
 #include "credentials.h"
 #endif
 
+unsigned int rotate = 0;
```

Comment:
But it needs to be outside of `setAllPicelColor` as this shifts every time you use the method. It could possibly be added in the `dimWithColor`-method instead of being in the global scope,

Reply:
It doesn't need to be outside if declared as `static`! :hatched_chick: